pants3

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+# This bootstrap script invokes Pants using a Python 3 interpreter.
+
+export PANTS_USE_PYTHON3=true
+./pants $@
```

Comment:
```suggestion
./pants "$@"
```

Without the quotes, if any of the args contain spaces, they will be split into separate args. Quotes around bash arrays does the thing you want :)